### PR TITLE
Redesigned stacks page launch

### PR DESCRIPTION
**Pull request by arunkumar611**
To-do
- Add video

### CODE REVIEW

File: content/blog/new-stacks-page-launch/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+<!--more-->
+
+## Why a New Stacks Page?
+
+As our customers scaled, the previous Stacks page faced some limitations that impacted its effectiveness. Here were some of the primary challenges:
+
+- **Performance Issues**: The page became sluggish with large numbers of stacks, making it difficult to scroll smoothly when you had thousands of stacks.
+- **Limited Grouping and Sorting Options**: Users had limited customization options for organizing their stacks, with minimal control over grouping and sorting.
+- **Tag-Based Grouping Limitations**: Grouping by tags wasn’t as reliable as we wanted—it often failed to load completely.
```
Comment (cleverguy25) [inline content/blog/new-stacks-page-launch/index.md:27]:
```suggestion
- **Tag-Based Grouping Limitations**: Grouping by tags wasn’t as reliable as we wanted—it sometimes failed to load completely.
```

File: content/blog/new-stacks-page-launch/index.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## What’s New?
+
+The new Stacks page features a modernized UI that provides more information at a glance while tackling these challenges head-on, offering a streamlined and powerful experience:
+
+- **Flexible Grouping**: Group your stacks by repository, project, and tags simultaneously—up to three levels deep. Or opt for no grouping at all! The choice is yours. Expand and collapse groups as needed for a more organized and focused browsing experience.
+- **Enhanced Sorting**: Sort your stacks by last update or name (ascending/descending) to quickly find what you need.
```
Comment (cleverguy25) [inline content/blog/new-stacks-page-launch/index.md:34]:
```suggestion
- **Enhanced Sorting**: Sort your stacks by last update or group name (ascending/descending) to quickly find what you need.
```

**Review by meagancojocar — APPROVED** [review]
LTGM!

**Review by arunkumar611 — COMMENTED** [review]
Changes from review

**Pull request merged**